project/src/com/google/daggerqueryui/scripts/analyze_input_field_content.js

```diff
@@ -91,6 +91,20 @@ $(function () {
       url.searchParams.append('query', query.join(' '));
       const graph = await $(this).getQueryResults(url);
       $(this).markInputFieldAsValid();
+
+      if (query[0] === $.DEPS_QUERY_NAME) {
+        bindingGraph.clear();
+        for (let childNode of graph) {
```

Comment:
nit: since `childNode` is never mutated, you can use `const` instead of `let`.

Reply:
Thanks, that's right ✅